src/components/Footer/MailchimpForm.tsx

```diff
@@ -3,13 +3,7 @@ import MailchimpSubscribe, {
   EmailFormFields,
   FormHooks,
 } from "react-mailchimp-subscribe"
```

Comment:
I believe we can remove these imports and `react-mailchimp-subscribe` dependency.

Reply:
Fixed in 728c282e33e7d69c0017dac9e73efee6c7486701